src/pages/Detail/component/DetailNav.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function DetailNav() {
+  return (
+    <Nav>
+      {SUB_TITLE.map(title => {
+        const { id, name } = title;
+        return <SubTitle key={id + name}>{name}</SubTitle>;
```

Comment:
```suggestion
        return <SubTitle key={id}>{name}</SubTitle>;
```
id로도 충분히 유니크한 값이 되기 때문에 더해주지 않으셔도 됩니다!